examples/sampling/graphbolt/node_classification.py

```diff
@@ -200,6 +246,7 @@ def train(args, graph, features, train_set, valid_set, num_classes, model):
     )
 
     for epoch in tqdm.trange(args.epochs):
+        print("Training...")
```

Comment:
nit: likely not needed per the output in description.

Reply:
What about `print("Validating...")` ?

Reply:
Probably not needed as well, check other examples for the convention, let's try to be consistent.
https://github.com/dmlc/dgl/blob/master/examples/sampling/node_classification.py
https://github.com/dmlc/dgl/blob/master/examples/sampling/link_prediction.py